### PR TITLE
Form fields update

### PR DESCRIPTION
**Pull request by milos-aleksic**
Total time dedicated: 3h

Being this the first commit, there was an initial time invested in setting up the project, getting familiarized with the existing structure, forms extensions and its settings, before starting to add more content to it. The estimated invested time in creating the documentation itself is 1.5h.

In addition to the existing structure a new table column, called "Dynamic" was added, for the setting fields that were accepting dynamic content, using a neutral check icon to define those fields within the table. This was done as I considered that is useful information to be stated in the documentation itself and could help to understand all the options of the field.

Also, references to UIkit docs were added for input size, input width, button style and button size, but its difficult to see them visually as those concur with the `quoted` code.

I also noticed there is a too repetitive use of the word "Settings", and in the future a solution could be to separate, in this specific case, field settings and element settings, being the first one strictly related to the field within the form and the second one related to that element options.

### CODE REVIEW

File: src/pages/essentials-for-yoothemepro/forms/elements/fields.md [hunk 1]
```diff
@@ -235,35 +235,152 @@ Month input relies on the browser native implementation, the picker and icon sty
...
+
+| Setting | Description | Dynamic |
+| ------- | ----------- | :-----: |
+| **Upload to** | Defines a path where the file will be uploaded. | &#x2713; |
+| **Avoid filename collisions** | Defines if the filename should change on upload, if conflicting with existing files. The newly created filename will be the original name appended with `_{n}`, being `n` a secuancial number. |
+| **Allow multiple uploads** | Defines if multiple files can be selected for upload, by adding the attribute `multiple` to `<input type="file">`. |
+| **Filename** | Defines an optional name to be set to the the file, upon upload. |
+
+| Validation | Description | Dynamic |
+| ---------- | ----------- | :-----: |
```
Comment (miljan-aleksic) [inline src/pages/essentials-for-yoothemepro/forms/elements/fields.md:285]: Any reason for using `:-----:` instead of just `-----` ?

Comment (milos-aleksic) [reply src/pages/essentials-for-yoothemepro/forms/elements/fields.md:285]: It centers the content, in this case the check icon.

File: src/pages/essentials-for-yoothemepro/forms/elements/fields.md [hunk 2]
```diff
@@ -235,35 +235,152 @@ Month input relies on the browser native implementation, the picker and icon sty
...
+---
+
+## Upload Element
+
+The Upload element displays by default a `<input type="file">` HTML element and a button with text.
+
+| Setting | Description | Dynamic |
+| ------- | ----------- | :-----: |
+| **Upload to** | Defines a path where the file will be uploaded. | &#x2713; |
+| **Avoid filename collisions** | Defines if the filename should change on upload, if conflicting with existing files. The newly created filename will be the original name appended with `_{n}`, being `n` a secuancial number. |
```
Comment (Skullbock) [inline src/pages/essentials-for-yoothemepro/forms/elements/fields.md:280]: Typo here, 
`secuancial` should be 
sequential

Comment (milos-aleksic) [reply src/pages/essentials-for-yoothemepro/forms/elements/fields.md]: Thanks, fixes committed on 8e4a933aaf5ecbc9040a435ef2a848b82565eea5

**Pull request merged**